lib/index.mjs

```diff
@@ -210,6 +211,10 @@ export async function checkNits (raw, filename, {
     result.push(...(await validateCodeBlockLicenses(doc, { mode })))
     progressReport('Validating updates and obsoletes...')
     result.push(...(await validateUpdatesAndObsoletesLines(doc, { mode })))
+    if (doc.docKind !== 'rfc') {
+      progressReport('Validating table of contents and document pages...')
```

Comment:
This isn't going to mean anything to the average document editor. I suggest "Verifying table of contents exists when required" instead.